snuba/subscriptions/scheduler_processing_strategy.py

```diff
@@ -45,12 +47,25 @@ class ProvideCommitStrategy(ProcessingStrategy[Tick]):
     guarantees that all subscriptions are scheduled at least once and we do
     not miss any even if the scheduler restarts and loses its state.
 
-    A tick can be safely committed only if its lower timestamp has also been
-    reached on every other partition. If that condition is not met, the message
-    is still submitted to the next step with a `should_commit` value of false
-    indicating that it's offset is not to be commited.
+    Each time a tick message is received by this strategy we need to figure out
+    if we can advance the offset that is committed. The offset that is considered
+    safe to commit is the latest offset that has been seen on every partition.
```

Comment:
Could you pelase specify here what you mean by `seen on every partition`.

Reply:
sorry i mean `latest offset that has been reached or exceeded on every partition`.. is that clearer?